templates/poster.html

```diff
@@ -1,4 +1,23 @@
 {% extends "base.html" %}
+
+{% block head %}
+
+{{ super() }}
+<meta name="citation_title" content="{{paper.content.title}}" />
+{% for author in paper.content.authors %}
+<meta name="citation_author" content="{{author}}" />
+{% endfor %}
+<meta name="citation_publication_date" content="{{config.date}}" />
+<meta name="citation_conference_title" content="{{config.tagline|striptags|title}}" />
+<meta name="citation_inbook_title" content="{{config.proceedings_title}}" />
+<meta name="citation_abstract" content="{{paper.content.abstract}}" />
+{% for keyword in paper.content.keywords %}
+<meta name="citation_keywords" content="{{keyword}}" />
+{% endfor %}
+<meta name="citation_pdf_url" content="{{paper.content.pdf_url | default("")}}" />
+
+{% endblock %}
```

Comment:
can you attach a screenshot of the poster page so we can see what it looks like?

Reply:
There aren't any viewable changes, but here's what the amended part of the head looks like: 
![image](https://user-images.githubusercontent.com/1337259/82840167-8d08ca00-9ec9-11ea-9fd1-6281339a1735.png)

Reply:
Ah I see. This is all meta data. Thanks!